docs/src/pages/guides/query-functions.md

```diff
@@ -8,12 +8,12 @@ A query function can be literally any function that **returns a promise**. The p
 All of the following are valid query function configurations:
 
 ```js
-useQuery(['todos', todoId], fetchTodoById)
```

Comment:
I would please keep one example that shows that the `queryFn` doesn't need to be an inline function. Maybe:

```
useQuery(['todos'], fetchAllTodos)
```

?

Reply:
good thinking — updated with your suggestion